src/ipc/ipc4/dai.c

```diff
@@ -298,8 +298,8 @@ int dai_config(struct comp_dev *dev, struct ipc_config_dai *common_config,
 	    dd->ipc_config.type != common_config->type)
 		return 0;
 
-	comp_info(dev, "dai_config() dai type = %d index = %d dd %p",
-		  common_config->type, common_config->dai_index, dd);
+	comp_dbg(dev, "dai_config() dai type = %d index = %d dd %p",
```

Comment:
please keep this one

Reply:
> 

Yes, I just put it here to let people know there is max performance data for first second, after that, max will return to normal, like below:

[   16.460291] <inf> ll_schedule: ll timer avg 2576, max 3136, overruns 0
[    9.485483] <inf> ll_schedule: ll timer avg 2523, max 3084, overruns 0

Reply:
@btian1 so this specific log can affect performance?